packages/terra-data-grid/tests/wdio/worklist-data-grid-spec.js

```diff
@@ -498,5 +498,51 @@ Terra.describeViewports('WorklistDataGrid', ['medium', 'large'], () => {
       expect($('tr.column-header-row').$('//th[1]').getCSSProperty('width').parsed.value).toBe(180);
       Terra.validates.element('column-resize-decrease-keyboard', options);
     });
+
+    it('focuses on the resize handle in the column header when navigating right from the column header', () => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/data-grid/worklist-data-grid/default-worklist-data-grid');
+
+      browser.keys(['Tab', 'ArrowRight']); // Focus is on (0,0)
+      expect(browser.$('tr.column-header-row th:nth-child(1) div[draggable=true]').isFocused()).toBe(true);
```

Comment:
Should we verify by the resize handle class name instead of depending on a draggable attribute being present?